### PR TITLE
#224 - extract repeating code to methods

### PR DESCRIPTION
**Pull request by olenagerasimova**
For #224 I've extracted repeating to methods.

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#225](https://github.com/artipie/rpm-adapter/issues/225) is assigned to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/915904a5-75de-4b5a-b003-619b851f81a6); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: src/main/java/com/artipie/rpm/Rpm.java [hunk 1]
```diff
@@ -291,31 +247,71 @@ public Completable updateBatchIncrementally(final Key prefix) {
...
-            ).map(path -> String.format("repodata/%s", path.getFileName().toString()))
-            .toList().map(HashSet::new).flatMapCompletable(
-                preserve -> new RxStorageWrapper(this.storage).list(new Key.From("repodata"))
-                    .flatMapObservable(Observable::fromIterable)
-                    .filter(item -> !((Set<String>) preserve).contains(item.string()))
-                    .flatMapCompletable(
-                        item -> new RxStorageWrapper(this.storage).delete(item)
-                    )
-            ).doOnTerminate(
+            .doOnNext(file -> Files.move(file, tmpdir.resolve(file.getFileName())))
```
Comment (carlosmiranda) [inline src/main/java/com/artipie/rpm/Rpm.java:250]: @olenagerasimova why did we remove `REPLACE_EXISTING` flag?

**Comment by olenagerasimova** [comment]
@carlosmiranda thank you for noticing, we actually need `replace` for the case when we are updating the same repository for the second time.

**Comment by olenagerasimova** [comment]
@carlosmiranda ping

**Comment by 0crat** [comment]
Job was finished in 47 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by sereshqua** [comment]
@carlosmiranda please make sure you will find at least 3 issues during next CR, thanks!

**Comment by sereshqua** [comment]
@carlosmiranda ping

**Comment by carlosmiranda** [comment]
@sereshqua confimed, thanks!

**Comment by sereshqua** [comment]
@0crat quality acceptable

**Pull request merged**